### PR TITLE
Isv-User manage service_templates valid namespace

### PR DESCRIPTION
**Pull request by baixinsui**
fixed https://github.com/eclipse-xpanse/xpanse/issues/791

### CODE REVIEW

File: modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java [hunk 1]
```diff
@@ -193,14 +202,24 @@ public ServiceTemplateEntity registerServiceTemplateByUrl(String oclLocation) th
...
             String errMsg = String.format("Service template with id %s not found.", id);
             log.error(errMsg);
             throw new ServiceTemplateNotRegistered(errMsg);
         }
+        if (checkNamespace) {
+            CurrentUserInfo currentUserInfo = identityProviderManager.getCurrentUserInfo();
+            if (Objects.nonNull(currentUserInfo)) {
+                if (!StringUtils.equals(serviceTemplateEntity.getNamespace(),
+                        currentUserInfo.getNamespace())) {
+                    throw new AccessDeniedException("No right to view details of service template "
```
Comment (swaroopar) [inline modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java:218]: No permissions to view details of services templates belonging to other namespaces.

File: modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java [hunk 2]
```diff
@@ -85,6 +86,14 @@ public ServiceTemplateEntity updateServiceTemplate(String id, Ocl ocl) {
             log.error(errMsg);
             throw new ServiceTemplateNotRegistered(errMsg);
         }
+        CurrentUserInfo currentUserInfo = identityProviderManager.getCurrentUserInfo();
+        if (Objects.nonNull(currentUserInfo)) {
+            if (!StringUtils.equals(existedService.getNamespace(),
+                    currentUserInfo.getNamespace())) {
+                throw new AccessDeniedException("No right to update service template belong to "
```
Comment (swaroopar) [inline modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java:93]: No permissions to update services templates belonging to other namespaces.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java (hunk 2 above)
Comment (baixinsui) [reply modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java]: Done.

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**